### DISCUSSION TITLE
Welcome to termux-pkg-manager Discussions!

### DISCUSSION THREAD

**Discussion by JesusChapman** [JesusChapman/termux-pkg-manager#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesusChapman** [comment]
@suhan-paradkar

It is very rare, tkinter did not load me the images, PySimpleGUI if it does...

**Reply by creepy-pasta101** [reply]
Did you use pillow??

**Reply by creepy-pasta101** [reply]
Also... you need to include the non-python files... i.e, the images in setup.py

**Comment by JesusChapman** [comment]
If I did, pillow didn't help me either, and PySimpleGUI if I charge images without using pillow

**Comment by JesusChapman** [comment]
I think it's better to use PySimpleGUI now, because of its simplicity and compatibility

**Comment by Pivete1533** [comment]
Ola
